Cargo.toml

```diff
@@ -1388,6 +1388,16 @@ zeroize = { version = "1.7.0", default-features = false }
 zombienet-sdk = { version = "0.2.13" }
 zstd = { version = "0.12.4", default-features = false }
 
+[patch.crates-io]
+
+# Patch away `libp2p-identity` in our dependency tree with the git version.
+# For details see: https://github.com/autonomys/rust-libp2p/blob/04c2e649b1f5482b8c3466b0fdbd1815b3126a48/Cargo.toml#L140-L145
+libp2p-identity = { git = "https://github.com/autonomys/rust-libp2p", rev = "04c2e649b1f5482b8c3466b0fdbd1815b3126a48" }
+
+# Patch away inconsistent transitive dependencies of libp2p
+multistream-select = { git = "https://github.com/autonomys/rust-libp2p", rev = "04c2e649b1f5482b8c3466b0fdbd1815b3126a48" }
+rw-stream-sink = { git = "https://github.com/autonomys/rust-libp2p", rev = "04c2e649b1f5482b8c3466b0fdbd1815b3126a48" }
```

Comment:
Why are transitive dependencies of the wrong version end up here?

Reply:
Probably for similar reasons to libp2p-identity: https://github.com/autonomys/rust-libp2p/blob/04c2e649b1f5482b8c3466b0fdbd1815b3126a48/Cargo.toml#L140-L145

Reply:
But they are not patched upstream :thinking: Check what pulls incorrect versions. I suspect it might be `litep2p`, in which case override is incorrect and should not be done since it might be a breaking change for litep2p that will be quite confusing to debug.

Reply:
It's `polkadot-zombienet-sdk-tests`, so it's irrelevant in production (but also not needed).